### DISCUSSION TITLE
Gemini Pro GCP Vertex support

### DISCUSSION THREAD

**Discussion by shing19** [langgenius/dify#3303] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Hi there! Gemini 1.5 Pro has released. It is really good. I would like to know if there are any plans to use the Gemini Pro model enterprise version of Google Cloud Vertex or if there is already a way to use it.

### 2. Describe the feature you'd like to see

to integrate models from GCP Vertex provider 

### 3. How will this feature improve your workflow or experience?

Gemini 1.5 Pro is powerful and better on wording style than GPT4. Highly recommend this one. 

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by ciekawy** [comment]
also gecko seems strong for embeddings, including multilingual

**Comment by patryk20120** [comment]
I am currently working on a new model provider for Vertex AI, hope this will help someone :)

**Comment by patryk20120** [comment]
https://github.com/langgenius/dify/pull/4586
